source/Generic/ExtraMetadataToolsVideo/ExtraMetadataToolsVideo.psm1

```diff
@@ -59,15 +59,241 @@ function GetGameMenuItems
+
+            <DockPanel>
+                <DockPanel.Style>
+                    <Style TargetType="DockPanel">
+                        <Setter Property="IsEnabled" Value="False" />
+                        <Style.Triggers>
+                            <DataTrigger Binding="{Binding SelectedValue, ElementName=CBqualityMode}" Value="manual">
+                                <Setter Property="IsEnabled" Value="True"/>
+                            </DataTrigger>
+                        </Style.Triggers>
+                    </Style>
+                </DockPanel.Style>
+
+                <DockPanel Margin="0,0,0,10" DockPanel.Dock="Top">
+                    <TextBlock Text="{DynamicResource LOCExtra_Metadata_tools_FILTER_FORMAT}" Name="TBLfilterFormat" DockPanel.Dock="Top" Margin="0,0,0,3"/>
+                    <TextBox Name="TBfilterFormat"/>
+                </DockPanel>
+
+                <DockPanel Margin="0,0,0,10">
+                    <TextBlock Text="{DynamicResource LOCExtra_Metadata_tools_SORT_FORMAT}" Name="TBLsortFormat" DockPanel.Dock="Top" Margin="0,0,0,3"/>
+                    <TextBox Name="TBsortFormat"/>
+                </DockPanel>
+            </DockPanel>
+        </StackPanel>
+    </DockPanel>
+</Grid>
+"@
+
+    # Load the xaml for controls
+    $XMLReader = [System.Xml.XmlNodeReader]::New($Xaml)
+    $XMLForm = [Windows.Markup.XamlReader]::Load($XMLReader)
+
+    # Make variables for each control
+    $Xaml.FirstChild.SelectNodes("//*[@Name]") | ForEach-Object {Set-Variable -Name $_.Name -Value $XMLForm.FindName($_.Name) }
+
+    # Set items sources of controls
+    $TBffmpegPath.Text = $settings.ffmpegPath
+    $TBffprobePath.Text = $settings.ffprobePath
+    $TBytdlpPath.Text = $settings.ytdlpPath
+    $TBthrottleLimit.Text = $settings.throttleLimit
+    $TBfilterFormat.Text = $settings.filterFormat
+    $TBsortFormat.Text = $settings.sortFormat
+
+    # Set combobox source
+    [System.Collections.ArrayList]$QualityModeSource = @(
+        [PSCustomObject]@{
+            Name = "yt-dlp Default (up to 720p)"
+            Value = "default"
+            Filter = "best"
+            Sort = ""
+        },
+        #Default preset
+        [PSCustomObject]@{
+            Name = "Better (up to 1080p)"
+            Value = "better"
+            Filter = "bv[width<=1920][height<=1080]+ba/b"
+            Sort = "lang,ext,quality,fps,size"
+        },
+        [PSCustomObject]@{
+            Name = "Best"
+            Value = "best"
+            Filter = "bv+ba/b"
+            Sort = "lang,ext,quality,fps,size"
+        },
+        [PSCustomObject]@{
+            Name = "Manual"
+            Value = "manual"
+            Filter = ""
+            Sort = ""
+        }
+    )
+
+    #Get selected preset by comparing filter + sort
+    $selectedList = $QualityModeSource | Where { $_.Filter -eq $settings.filterFormat } | Where { $_.Sort -eq $settings.sortFormat } 
```

Comment:
Same, change the `Where` alias to the full function name `Where-Object` please

Reply:
Done in 30b385caf6ed9eef73af37eaed397e5b316ceced